wandb/apis/public/files.py

```diff
@@ -116,7 +117,7 @@ class File(Attrs):
         mimetype (string): mimetype of file
         updated_at (string): timestamp of last update
         size (int): size of file in bytes
-
+        path_uri (str): path to file in the bucket, currently only available for artifacts stored in S3
```

Comment:
Since we're adding this attribute to the file class, do you think it'll be applicable to RunFiles too?

Reply:
so yes this works for run files as well, I'll change the doc strings. @noaleetz should be fine that run files also get this functionality right?